packages/components/pie-card/src/index.ts

```diff
@@ -66,13 +66,13 @@ export class PieCard extends LitElement implements CardProps {
                 ?isDraggable="${this.isDraggable}"
                 variant=${this.variant || 'default'}
                 ?disabled=${this.disabled}
-                href=${this.href || nothing}
+                href=${this.href || ''}
                 target=${this.target || nothing}
                 rel=${this.rel || nothing}
                 role="link"
                 aria-label=${this.aria?.label || nothing}
                 aria-disabled=${this.disabled ? 'true' : 'false'}
-                style=${paddingCSS || nothing}>
+                style=${paddingCSS || ''}>
```

Comment:
Would this be able to use ifDefined() out of interest that way it doesn't render an empty style tag?